src/thor/bidirectional_astar.cc

```diff
@@ -754,7 +787,7 @@ std::vector<PathInfo> BidirectionalAStar::FormPath(GraphReader& graphreader) {
       cost += edgelabel.cost() - edgelabels_reverse_[predidx].cost();
     }
     cost += tc;
-    path.emplace_back(edgelabel.mode(), cost.secs, oppedge, 0, cost.cost);
+    path.emplace_back(edgelabel.mode(), cost.secs, edgelabel.opp_edgeid(), 0, cost.cost);
```

Comment:
this was a simplification. notice above that we go fetch the opposing edge id the hard way. the edge labels for bidirectional already have the opposing edge id on them so i just used that instead.  @dnesbitt61 there's not a gotcha here right?